crates/biome_css_parser/tests/css_test_suite/ok/scss/declaration/valid-modern-values.scss

```
@@ -0,0 +1,33 @@
+$spacing: 1rem !default;
+$theme-colors: (
+  primary: #0a84ff,
+  accent: color.adjust(#0a84ff, $lightness: 10%),
+);
+$font-stack: (system-ui, "Helvetica Neue", sans-serif);
+
+.foo {
+  --gap: 10px;
+  --#{$prop}: 10px;
+  --theme-#{$slot}: color.adjust(#fff, $alpha: -0.2);
+  #{$prop}: var(--theme-#{$slot});
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)